### ISSUE TITLE
anucleate and binucleate compartment

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#2572]
I know we have been here before but at least I know which terms to use now, so we have moved forward some.

The only difference I see between
FYPO:0003166	septation following abnormal chromosome segregation, with binucleate and anucleate compartment formation

and
FYPO:0004699	monoseptate vegetative cell with binucleate and anucleate compartments

is that 
FYPO:0004699 specifies only a single septa?

In which case I can't figure out why  FYPO:0003166 has more annotations?

![query terms numbers](https://cloud.githubusercontent.com/assets/7359272/13458322/7a3f6d98-e064-11e5-8960-4f3269f00094.jpg)

also,  isn't
FYPO:0004699	monoseptate vegetative cell with binucleate and anucleate compartments
always also following abnormal chromosome segregation?

**Comment by ValWood** [comment]
note to self, fix 37cb96e688f6a42c when I'm sure

**Comment by mah11** [comment]
Looking at the names, defs, etc. it actually seems like FYPO:0003166 is trying to say more about how cells got that way. But maybe we don't need that. If you just want to annotate what you see, then the terms could probably be merged. (FYPO:0003166 has a synonym with "monoseptate", so we must not have been consciously thinking of the possibility of multiple septa.)

I guess FYPO:0003166 has more annotations simply because it's been around a year and a half longer. It may also be a bit easier to find, since it's the sibling of cut.

tl;dr: no prob to merge if you want

**Comment by ValWood** [comment]
Lets merge them. Its the same observation.

**Comment by mah11** [comment]
OK. Do you have a preference which name we use?

**Comment by ValWood** [comment]
monoseptate vegetative cell with binucleate and anucleate compartments
sounds better?

**Comment by mah11** [comment]
OK, merged, using the id FYPO:0003166 but the name from FYPO:0004699

**Comment by mah11** [comment]
edit file: 138d806d362c771009d917f669073b54cb4452af
release: 0c9f107ee56df966ee46641cac1897aa7fa31157
